### ISSUE TITLE
Adding specifiers

### ISSUE THREAD

**Issue by jayfk** [vstinner/python-security#1]
Hey, thanks for your work on this @haypo.

Would you consider adding `SpecifierSets` of vulnerable releases to your entries in https://github.com/haypo/python-security/blob/master/vulnerabilities.yaml?

This would make it much easier for automated tools to read what exactly is vulnerable.

I'd be happy to open a PR for this.

**Comment by jayfk** [comment]
Ah okay, I wasn't quite sure what `render_doc.py` actually does. My initial thought was that it takes `vulnerabilities.yml` as an input source and builds a doc from it.

> Can you give an example of what you expect?

Ideally, I'd like to have a way to point to all releases that are vulnerable to a particular bug. Looking at http://python-security.readthedocs.io/vulnerabilities.html the only thing that seems to be missing is where the bug was introduced.

Let's take http://python-security.readthedocs.io/vuln/issue_28563_gettext.c2py.html#issue-28563-gettext-c2py as an example:
 * I'm unable to tell if Python 2.7.2 is affected. All I know that it was fixed in Python 2.7.12. Maybe it was introduced in Python 2.7.3, so Python 2.7.2 is not affected.
 * I know that 3.3 is vulnerable, because it wasn't fixed yet. Is this true for all of 3.3.x?

Specifiers are a great way to make this visible, but I'm happy with everything that allows me to compute this on my own.

Given the example from above, the specifiers would be:
```
>=2.7,<2.7.13 # Python 2.7.13
>=3.3,<3.4 # all of 3.3 is affected
>=3.4,<3.4.6 # Python 3.4.6
>=3.5,<3.5.3 # Python 3.5.3
>=3.6,<3.6.0 # Python 3.6.0, this allows to see that alpha/beta/rc releases are vulnerable
```

If the bug was indeed introduced in 2.7.3, the specs would change to:

```
>=2.7.3,<2.7.13
```

When looking at https://github.com/haypo/python-security/blob/master/render_doc.py#L544, there's probably no way to compute where a bug was introduced correctly, the data just isn't there.

**Comment by vstinner** [comment]
This issue didn't get any activity for longer than 1 year, I close it.
